src/components/cart/cart-item-options/cart-item-options.module.scss

```diff
@@ -9,7 +9,6 @@
     align-items: center;
     gap: 6px;
     font: var(--paragraph3);
-    cursor: pointer;
```

Comment:
I haven't verified everything, but there's probably the same issue with cursor removal everywhere.